File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Model/src/Team.cs
namespace Ubb.BikeContest.Model;

[Serializable]
public class Team : Identifiable<long>
{
    public Team(string name)
    {
        Name = name;
    }
    
    public string Name { get; set; }

    public override string ToString()
    {
        return Name;
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab09/model/src/main/java/com/ubb/model/Team.java
package com.ubb.model;

public class Team extends Identifiable<Long> {
    private final String name;

    {
        setId(0L);
    }

    public Team(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    @Override
    public String toString() {
        return getName();
    }
}
